src/modules/science/index.ts

```diff
@@ -0,0 +1,58 @@
+import type { Faker } from '../..';
+
+export type Element = {
+  name: string;
+  symbol: string;
+};
+
+export type Unit = {
+  long: string;
+  short: string;
+};
+
+/**
+ * Module to generate science related entries.
+ */
+export class Science {
+  constructor(private readonly faker: Faker) {
+    // Bind `this` so namespaced is working correctly
+    for (const name of Object.getOwnPropertyNames(Science.prototype)) {
+      if (name === 'constructor' || typeof this[name] !== 'function') {
+        continue;
+      }
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Returns a random element.
+   *
+   * @example
+   * faker.science.element() // { name: 'Hydrogen', symbol: 'H' }
+   * faker.science.element() // { name: 'Xenon', symbol: 'Xe' }
+   * faker.science.element() // { name: 'Cerium', symbol: 'Ce' }
+   */
+  element(): Element {
+    const index = this.faker.datatype.number({
+      min: 0,
+      max: this.faker.definitions.science.chemicalElement.length - 1,
+    });
+    return this.faker.definitions.science.chemicalElement[index];
```

Comment:
Why don't you use `helpers.arrayElement`?
```suggestion
    return this.faker.helpers.arrayElement(this.faker.definitions.science.chemicalElement);
```